onnxruntime/contrib_ops/cpu/quantization/matmul_nbits.cc

```diff
@@ -239,6 +267,28 @@ Status MatMulNBits<MLFloat16>::PrePack(const Tensor& tensor, int input_idx, /*ou
 #endif  // MLAS_TARGET_AMD64_IX86
   }
 
+  if (save_prepacked_initializers) {
+    ConvertPrepackWeightIntoTensor(tensor, input_idx);
+  }
+
+  return Status::OK();
+}
+
+template <typename T1>
+Tensor* MatMulNBits<T1>::GetPrePackTensors() {
```

Comment:
Generally, kernels do not use Tensors to store pre-packed weights. Plus, there might be several of them.
I was under impression that a kernel would
a) return Tensors by value, because they are needed for external files.
b) there could be many of them.

What am I missing?